package.json

```diff
@@ -14,7 +14,8 @@
     "lint:fix": "npm run lint --silent -- --fix",
     "lint:watch": "watch 'npm run lint --silent' src",
     "postinstall": "npm run build",
-    "test": "ava"
+    "test": "ava",
+    "test:watch": "watch 'npm test --silent' src test"
```

Comment:
Used `watch` here over the built in watch so we can take advantage of the `--silent` flag.  Also, the `ava` watch wasn't actually working.